src/env.cc

```diff
@@ -644,7 +644,7 @@ Environment::Environment(IsolateData* isolate_data,
       exec_argv_(exec_args),
       argv_(args),
       exec_path_(GetExecPath(args)),
-      exiting_(isolate_, 1, MAYBE_FIELD_PTR(env_info, exiting)),
+      exit_info_(isolate_, 3, MAYBE_FIELD_PTR(env_info, exit_info)),
```

Comment:
nit: maybe we can add another value at the end of the ExitInfoField enum and name it something like ExitInfoFieldCount and use that here instead of 3? We do that in a couple of places like https://github.com/nodejs/node/blob/7124476fb3a9a58fd316f9cef3329c14fc1cd966/src/base_object.h#L47 for example. The advantage is that we wouldn't have to update this value in case a new value is added to the enum in the future.

Reply:
Applied.